x/supplier/types/key_proof.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import "encoding/binary"
+
+var _ binary.ByteOrder
+
+const (
+	// ProofKeyPrefix is the prefix to retrieve all Proof
+	ProofKeyPrefix = "Proof/value/"
+)
+
+// ProofKey returns the store key to retrieve a Proof from the index fields
+func ProofKey(
```

Comment:
Do we need to leave a `// TODO_TECHDEBT` here regarding updating the key/index type or constructor?